packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-container.js

```diff
@@ -12,6 +12,7 @@ registerStyles(
     [part='wrapper'] {
       display: flex;
       width: 100%;
+      min-width: 0;
```

Comment:
This is needed to ensure wrapper does not get too wide (especially when using Material theme).